test/tbb/test_partitioner.cpp

```diff
@@ -1,5 +1,5 @@
 /*
-    Copyright (c) 2021 Intel Corporation
+    Copyright (c) 2022 Intel Corporation
```

Comment:
Why replacing? Isn't it should be extended instead?
```suggestion
    Copyright (c) 2021-2022 Intel Corporation
```

Reply:
Agree with @aleksei-fedotov . While changing the existing file the year range should be used.